web/src/demo_components/button_group.dart

```diff
@@ -1,4 +1,8 @@
-part of over_react.web.demo_components;
```

Comment:
What's the reason to split this lib into multiple libs?

Reply:
just got tired of the parts, they are kind of an anti-pattern.